spring-cloud-gcp-secretmanager/src/test/java/org/springframework/cloud/gcp/secretmanager/SecretManagerTemplateTests.java

```diff
@@ -90,6 +90,35 @@ public void testCreateSecretIfMissing() {
 		verify(this.client).addSecretVersion(addSecretVersionRequest);
 	}
 
+	@Test
+	public void testCreateSecretIfMissingWithProject() {
```

Comment:
```suggestion
	public void testCreateSecretIfMissing_withProject() {
```

Minor style - if you could make modify the test name to be `_withProject` I think it is more clear that it refers to using the project parameter

Reply:
done in newer commit